src/cloud_common/protocols/unit/unit_config.py

```diff
@@ -159,12 +172,12 @@ class AlertRules(BaseModel):
         ),
     ]
 
-    storage: Annotated[
-        AlertRulePercents,
+    used_disks: Annotated[
```

Comment:
Let's name it just `disk` to match same in the monitoring struct.

Reply:
updated. done